docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,1981 @@
+* Do not update defaultIndex in case of insufficient permissions {kibana-pull}134202[#134202]
+* Fixes flaky test for "allows editing of a newly created field" {kibana-pull}132812[#132812]
+* Fixes flaky test for "context encoded URL params" {kibana-pull}132808[#132808]
+
+Elastic Security::
+For the Elastic Security 8.4.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+Using point in time for agent status query to avoid discrepancy {kibana-pull}135816[#135816]
+
+Lens & Visualizations::
+* Make reference line chart more robust in *Lens* {kibana-pull}137101[#137101]
+* Format the label with the right default formatter in *TSVB* {kibana-pull}136934[#136934]
+* Removes extra space from the legend when it is positioned on top/bottom {kibana-pull}135982[#135982]
+* Display Y-axis tick labels {kibana-pull}135976[#135976]
+* Fixes Date histogram bounds calculation doesn't update "now" {kibana-pull}135899[#135899]
+* Fixes internal links in *Vega* {kibana-pull}135890[#135890]
+* Do not set non-unique id for legend action popovers in *Lens* {kibana-pull}135656[#135656]
+* Fixes non-editable Lens panel when using prefix wildcard in *Lens* {kibana-pull}135654[#135654]
+* Removes saved search references from all places on unlink in *Visualize* {kibana-pull}135628[#135628]
+* Fix multi-field top values for Heatmap visualizations in *Lens* {kibana-pull}135581[#135581]
+* Fixes styling issues in *Lens* {kibana-pull}135406[#135406]
+* Show badge for read-only in *Lens* {kibana-pull}135313[#135313]
+* Don't let reference line fills on different axes collide in *Lens* {kibana-pull}135299[#135299]
+* Fixes css specificity issue in *TSVB* {kibana-pull}135245[#135245]
+* Always show palette on first dimension for mosaic in *Lens* {kibana-pull}135198[#135198]
+* Wrong `visType` for `horizontal_bar` visualization {kibana-pull}135013[#135013]
+* Unlinking Some Agg Based Visualizations Results in Unsaved Changes in *Visualize* {kibana-pull}134229[#134229]
+* [Datatable] Ellipsis truncation is not visible in cells. Letters are just cut off. in *Lens* {kibana-pull}134065[#134065]
+* Switching dashboard mode doesn't update missing data view prompt in *Visualize* {kibana-pull}133873[#133873]
+* Fixes application of suffix formats in *Lens* {kibana-pull}133780[#133780]
+* Error messages not centered in *TSVB* {kibana-pull}133288[#133288]
+* Use correct time zone for time shifting {kibana-pull}133141[#133141]
+
+Machine Learning::
+* Lock the delete annotation button on click {kibana-pull}137306[#137306]
+* Fixes globally pinned filters in Data visualizer and query search bar not clearing properly for saved searches {kibana-pull}136897[#136897]
+* Fixes overflow in start datafeed modal {kibana-pull}136292[#136292]
+* Fixes error in categorization wizard summary step {kibana-pull}134228[#134228]
+
+Management::
```

Comment:
There are a few extra transforms fixes that have been merged in the past week:
https://github.com/elastic/kibana/pull/137773
https://github.com/elastic/kibana/pull/138630
https://github.com/elastic/kibana/pull/138756